.env

```
@@ -0,0 +1,7 @@
+# NOTE
+# You SHOULD specify point releases here so that build time and run time Erlang/OTPs
+# are the same. See: https://github.com/erlware/relx/pull/902
+SERVICE_NAME=capi_pcidss
```

```suggestion
SERVICE_NAME=capi-pcidss
```

Чтобы назывался (почти) так же, как репозиторий и image tag.

---

Тогда при выкатке надо не забыть поменять везде пути 🤔

---

Это да. 😩